android/com/facebook/buck/android/support/exopackage/DefaultApplicationLike.java

```diff
@@ -42,4 +42,9 @@ public void onTerminate() {}
 
   @Override
   public void onConfigurationChanged(Configuration newConfig) {}
+
+  @Override
+  public Object getSystemService(String name) {
```

Comment:
This module has no dependencies on those annotations. I'm not sure we want to add them, I'll leave that up to the buck maintainers to decide.